SMCalloutView.m

```diff
@@ -159,6 +159,9 @@ - (CGFloat)leftAccessoryVerticalMargin {
         return 0;
 }
 
+#pragma clang diagnostic push
```

Comment:
When the compiler is pedantic, it’s really pedantic. The problem is that `MIN()` and `MAX()` are [implemented using the `({…})` GNU extension](http://stackoverflow.com/questions/13087034/warning-use-of-gnu-statement-expression-extension/13087148#13087148).